tests/workflow_tests/test_deployment_update.py

```diff
@@ -2004,3 +2012,104 @@ def test_remove_deployment(self):
         self.assertEquals(len(deployment_update_list), len(undel_depups_ids))
         for i in deployment_update_list.items:
             self.assertIn(i['id'], undel_depups_ids)
+
+
+class TestDeploymentUpdateMisc(DeploymentUpdateBase):
```

Comment:
well, maybe the name is misleading, but the mixoperations basically held any test which is more than one step/skipping install/executing custom workflow etc...

it sounds like it's a good idea to extract any non step related test to your new class...